### DISCUSSION TITLE
Seeking to install of version of MOOSE on a PC with Windows-7 (64-bit) as the OS

### DISCUSSION THREAD

**Discussion by steinfel00physicer** [idaholab/moose#16108] [Q&A Getting Started] [answered]
I wish to have my question answered. I see that a Windows 10 version of MOOSE is available to install. 

Is it possible to install MOOSE (windows-10 version) onto a Computer which has Windows-7 rather than Windows-10 for its O.S. ?        Or is a download of a Windows-7 version of MOOSE available to install ?

**Comment by crswong888** [comment]
I'm pretty sure our first ever PC package was for Windows 10. So I'm going to guess that there is no compatibility for any earlier versions. Give [Ubuntu](https://ubuntu.com/) a try - it's free and super easy to use, although setting up the dual-boot tends to be tricky if you've never done that sort of thing before. But I could help with that if you need it.

**Comment by cticenhour** [comment, marked as answer]
MOOSE on Windows currently relies on Windows Subsystem for Linux, which is a Windows 10-only feature. There are currently no plans to bring a native developer environment for MOOSE to Windows 7. There was some work to make a native executable for Windows 10 related to #14591, but as the developer in that issue stated, this is not suitable for a developer environment. If you wish to remain on Windows 7 (not recommended since you open yourself up to security risks now that it is no longer supported by Microsoft), you can use a virtual machine program (such as VirtualBox) to run Linux inside of your Windows instance.
